### PR TITLE
#46 Java8 docker container for rultor

### PR DESCRIPTION
**Pull request by longtimeago**

### CODE REVIEW

**Comment by longtimeago** [comment]
@karato please, assign someone for review

**Comment by karato** [comment]
Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@darkled review it

File: .rultor.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ install: |-
   sudo apt-get install bsdmainutils
   sudo gem install pdd
   sudo gem install est
+docker:
+  image: mostalive/ubuntu-14.04-oracle-jdk8
```
Comment (asaen) [inline .rultor.yml]: @longtimeago please surround it with quote as mentioned in the issue, it should look like `image: "mostalive/ubuntu-14.04-oracle-jdk8"`

**Comment by asaen** [comment]
@longtimeago thanks, just a comment above

**Comment by longtimeago** [comment]
@darkled thanks, done

**Comment by asaen** [comment]
@longtimeago thanks

**Comment by asaen** [comment]
@rultor merge

**Comment by asaen** [comment]
@ggajos please, do not confirm it so far! We've found that this image doesn't have ruby and maven. So we'll be not able to merge any other PRs if we merge this one right now.

**Comment by longtimeago** [comment]
@darkled looks like we need to create out own image, with ruby and maven. @yegor256 What do you think?

**Comment by yegor256** [comment]
@longtimeago yes, we can create our own image OR we can just add Ruby and Maven installation instructions into `install` part of `.rultor.yml`:

``` yaml
install: |
  sudo apt-get install ruby
  sudo apt-get install maven
```

**Comment by longtimeago** [comment]
@yegor256 For now we can start with install section, but this approach will slow down each build. Creating the image is more preferable IMHO

**Comment by yegor256** [comment]
@longtimeago yeah, of course, but for now this will work. It will add a few seconds to each build. not a big deal, I think

**Comment by longtimeago** [comment]
@darkled @yegor256 PR was updated

**Comment by ggajos** [comment]
@rultor merge

**Comment by asaen** [comment]
@longtimeago I guess installation of ruby and maven should be done at the very beginning, it seems that `gem` util needs ruby

**Comment by karato** [comment]
@darkled thank you, added **18 mins** to your acc, payment num is `53344236`

extra minutes for review comments (c=3)

+18 added to your rating, current score is: [+1956](http://www.netbout.com/b/35091?open=rating)

**Comment by karato** [comment]
@rultor please deploy

**Pull request merged**